main.rb

```diff
@@ -6,6 +6,13 @@
 
 java_import "org.yaml.snakeyaml.Yaml"
 
+# solrj stuff
+java_import "org.apache.solr.client.solrj.SolrServerException"
+java_import "org.apache.solr.client.solrj.impl.HttpSolrServer"
```

Comment:
I think we would want EmbeddedSolrServer (https://lucene.apache.org/solr/5_5_0/solr-core/org/apache/solr/client/solrj/embedded/EmbeddedSolrServer.html) and CoreContainer (https://lucene.apache.org/solr/5_5_0/solr-core/org/apache/solr/core/CoreContainer.html) so that we can do something like Zest does (https://github.com/apache/zest-java/blob/f21cc656a72b6fb5fdca112b05133d7e63455a3d/extensions/indexing-solr/src/main/java/org/apache/zest/index/solr/EmbeddedSolrService.java#L98)